### PR TITLE
v84.6.0

### PR DESCRIPTION
**Pull request by codeman7**
#### Don't forget:
- [ ] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [ ] Sign the CLA bot. You can do this once the pull request is opened.

[Contributing](./contributing/README.md#pull-requests) has more information and tips for a great
pull request.

### CODE REVIEW

**Review by jverkoey — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
+# 84.6.0
+
+In this minor release we added new features to ActionSheet and NavigationDrawer, as well as additional tests and bug fixes.
+
+## New features
+
+ActionSheet now allows clients to align all titles regardless of if an image is present or not.
+
+```swift
```
Comment (romoore) [inline CHANGELOG.md:9]: Tip: Add 2 actions - one with title + image, one with title only.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+let actionSheet = MDCActionSheetController()
+actionSheet.alwaysAlignTitleLeadingEdges = true
+```
+
+NavigationDrawer now allows clients to include the safeAreaInsets to the content height.
```
Comment (romoore) [inline CHANGELOG.md:14]: Tip: Suggestion: "...to automatically include `safeAreaInsets` in the content height calculation."

Comment (romoore) [inline CHANGELOG.md:14]: Can you clarify what effect this has on the view controller in more technical terms? I don't see a `contentHeight` API.

**Review by romoore — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (codeman7) [reply CHANGELOG.md:9]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (codeman7) [reply CHANGELOG.md:14]: Done

Comment (codeman7) [reply CHANGELOG.md:14]: Done

**Pull request merged**